workflow/qa-steps.md

```diff
@@ -0,0 +1,35 @@
+# QA steps
+
+In this section you will find a list of steps to take when reviewing PRs, in order to catch unforeseen regressions.
+
+## Adding content
+
+- If adding a new page to a site, check that it has been added to the navigation, sitemap and breadcrumbs (where applicable)
+- If applicable, check that the new content's copy matches the relevant copydoc.
+- Conversely, update the copydoc if it is out of date with the content.
+
+## Removing a page
+
+- When removing a page, check that it has been removed from the navigation, sitemap and breadcrumbs (where applicable)
+- Ensure that redirects are in place for deprecated urls
+
+## Navigation
+
+- Check that clicking a navigation link takes you to the right place.
+- If applicable, check that the hover/active state on a navigation link still works.
+
+## Search
+
+- If the site has search functionality, check that searching works properly and you are given relevant results.
+
+## Forms
+
+- If the page you are QAing contains a form, fill it out and check that it still submits. Also, fill it out incorrectly and check that validation still works.
+
+## Browser testing
+
+- Check that the changes look correct on Chrome, Firefox and IE/Edge (see [Browser support](/coding/browser-support.md))
+
+## Responsiveness
+
+- Check that the changes look correct on mobile, tablet and desktop sizes.
```

Comment:
Could this point we placed under "Browser testing"? It doesn't seem to quite warrant another additional heading.

Reply:
I would defend the separate categorisation as they are different review aspects. For example, a reviewer should always check the different viewports on their default browser but it is not always necessary or reasonable to browser test all reviews.